File: Aptech/C language/C/C/functionsinc/Factorial.cpp
#include<stdio.h>
int fact(int a);

int main()
{
	int k,p;
	printf("Enter a number for finding factorial: ");
	scanf("%d",&k);
	p=fact(k);
	printf("Factorial of the given number %d  = %d",k,p);
}
int fact(int n)
{
	int b,result=1;
	for(b=1;b<=n;b++)
		result=result*b;
	return (result);
}
	

File: College/Ankan_JAVA/Assignment_One/Factorial.java
import java.util.Scanner;

public class Factorial {
    public static void main(String[] args) {
        int number, fact = 1, temp;
        try (Scanner input = new Scanner(System.in)) {
            System.out.println("Enter the number ");
            number = input.nextInt();
            for (temp = number; temp >= 1; temp--) {
                fact = fact * temp;
            }
            System.out.println("Factorial is  :  " + fact);
            input.close();
        }
    }
}
